.github/workflows/wheels.yml

```diff
@@ -0,0 +1,53 @@
+name: Wheels
+
+on: [push, pull_request]
```

Comment:
Do we want to trigger wheel build on all PRs and pushes?

Or we can just build wheels whenever a new release tag is created:

```yaml
on:
  push:
    tags:
      - 'v*.*.*'
 ```

Reply:
I asked same thing. that was to test this PR. I'll change to that after merge.

Reply:
sounds good